src/server/controllers/userController.js

```diff
@@ -46,3 +47,48 @@ userController.getUser = async (req, res, next) => {
   }
 };
 
+userController.addUser = async (req, res, next) => {
+  try {
+    const { name, username, password, instrumentation, videoURL, profilephoto_URL, location, availability, email, short_bio, socialmedia_link,  user_role } = req.body;
+    const addedUser = await model.User.create(req.body);
```

Comment:
if we can use create(req.body) directly, why don't we still want to destruct req.body